test/find-package-test/CMakeLists.txt

```diff
@@ -1,17 +1,37 @@
-cmake_minimum_required(VERSION 3.8...3.25)
+cmake_minimum_required(VERSION 3.8...3.30)
 
 project(fmt-test)
 
-find_package(FMT REQUIRED)
+find_package(FMT 11.0.0 REQUIRED HINTS ${FMT_DIR})
+
+if (CMAKE_VERSION VERSION_GREATER_EQUAL 3.28.4 AND
+    CMAKE_GENERATOR STREQUAL "Ninja" AND
+    CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
```

Comment:
The Clang version must be >= v17
I saw also problems with g++ v14 on OSX, I have to check it again.